docs/about/releases.mdx

```diff
@@ -14,8 +14,17 @@ description: "Latest new features and changelog."
 ---
 
 [All releases](https://github.com/mage-ai/mage-ai/releases)
+- [0.9.70 | Fallout Release](https://github.com/mage-ai/mage-ai/releases/tag/0.9.70)
+<iframe
+  width="560"
+  height="315"
+  src="https://youtu.be/rAp7psCTxC8?si=A4BtcYWuiQJF1R8x"
```

Comment:
@thomaschung408 You needa use the youtube embed link (e.g. `https://www.youtube.com/embed/ck1UGjUeGGA`), which has `/embed/` in the link.